.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Report a bug in the project
+title: "Bug Title"
+labels: ["kind/bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thank you for your report. Before you submit the issue please:
+        1. Search open and closed issues for duplicates.
+        2. Check the [Discussions](https://github.com/kyma-project/lifecycle-manager/discussions) and the [FAQ](https://github.com/kyma-project/lifecycle-manager/wiki/FAQ) sections of the Lifecycle Manager repository.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Provide a general description of the bug that you would like to report.
+      placeholder: Provide your bug description here.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: Provide a detailed list of steps required to reproduce this issue.
+      placeholder: Provide your list here.
+    validations:
+      required: true
+  - type: dropdown
+    id: env-type
+    attributes:
+      label: Environment Type
+      multiple: false
+      options:
+        - Managed
+        - minikube
+    validations:
+      required: true
+  - type: textarea
+    id: environment-info
+    attributes:
+      label: Environment Info
+      description: More information on the environment where the bug was found.
```

```suggestion
      description: More information on the environment where you found the bug.
```
The less passive and more active voice the better :)